azure-mgmt-monitor/src/main/java/com/microsoft/azure/management/monitor/EventData.java

```diff
@@ -0,0 +1,287 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ *
+ * Code generated by Microsoft (R) AutoRest Code Generator.
+ */
+
+package com.microsoft.azure.management.monitor;
+
+import com.microsoft.azure.management.apigeneration.LangDefinition;
+import com.microsoft.azure.management.apigeneration.LangMethodDefinition;
+import com.microsoft.azure.management.monitor.implementation.EventDataInner;
+import org.joda.time.DateTime;
+
+import java.util.Map;
+
+/**
+ * The Azure event log entries are of type EventData.
+ */
+@LangDefinition(ContainerName = "/Microsoft.Azure.Management.Monitor.Fluent.Models")
+public class EventData {
```

Comment:
Usually we have both interface and implementation classes for wrappers like this.

Reply:
please take a look at QueryKey.java and QueryKeyImpl.java under com.microsoft.azure.management.search

Reply:
fixed